app/main.py

```diff
@@ -1,8 +1,26 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
```

Comment:
The `Person.people` dictionary is updated with the new `Person` instance using the name as the key. This could lead to unexpected behavior if a `Person` object is created with a duplicate name, as it will overwrite the existing entry. Consider adding a check to ensure unique names or handle duplicates appropriately.